components/resourceDetails/resourceSharing/agentAccessTable/index.test.jsx

```diff
@@ -151,49 +232,60 @@ describe("AgentAccessTable", () => {
   });
   // TODO: this will change once we have groups and a new way of filtering
   it("renders a set of tabs which filter by type", () => {
+    const permissionsWithTypes = [
+      {
```

Comment:
Could these object be reused more easily by extracting them into mock functions, or just mocked objects somewhere?